web/ui/src/idai_field/project/ProjectHome.tsx

```diff
@@ -87,17 +89,20 @@ export default function ProjectHome(): ReactElement {
 }
 
 
-const renderTitle = (title: string, projectDocument: Document) =>
-    <div className="d-flex p-2 m-2" style={ headerStyle }>
-        <div className="flex-fill">
-            <h2><img src="/marker-icon.svg" alt="Home" style={ homeIconStyle } /> {title}</h2>
-        </div>
-        <div className="text-right" style={ buttonsStyle }>
-            <LicenseInformationButton license={ projectDocument.resource.license } />
-            <DocumentPermalinkButton url={ getDocumentPermalink(projectDocument) } />
-        </div>
-    </div>;
+const renderTitle = (title: string, projectDocument: Document) => {
+
+    const titleStr = getTranslation(title as undefined as I18N.String);
```

Comment:
fix typing. param `title` should be of type I18N.String.
auch nochmal gucken, wenn ich den rückgabewert vom getTranslation anpasse (siehe unten, '' vs undefined)